### DISCUSSION TITLE
How to use multiple classes for a control?

### DISCUSSION THREAD

**Discussion by XDgl233** [AvaloniaUI/Avalonia#12307] [Q&A] [answered]
I saw the example shown in [the Doc](https://docs.avaloniaui.net/docs/next/reference/styles/style-selector-syntax#by-style-class).

> <Style Selector="Button.large.red">

> Selects a control with the specified style class or classes. Multiple classes are separated with a full stop. If multiple classes are specified in the selector, then the control must have all of the requested classes defined for a match.

I am very interested in doing this way to manage my controls. But how do I implement this in the actual control, I tried to set up the style like the example, Button.large.red. And I use Classes = "large.red" in <Button> control, but no effect. And I saw some old discussions in 2020, using some classes like Classsetter, but I am currently working on Avalonia 11.0, and I can't find such a class.
Really appreciate and thanks for any help and input :)

Edit: Fix some typos.

**Comment by timunie** [comment, marked as answer]
in Styles:
```xml
<UserControl.Styles>
  <Style Selector="Button.large.red">
    <Setter Property="Background" Value="Red"></Setter>
    <Setter Property="FontSize" Value="25"></Setter>
  </Style>
</UserControl.Styles>
```

Usage is `<Button Classes="large red">Hello World</Button>`

**Reply by XDgl233** [reply]
Thank you so much! :)
